### PR TITLE
NTR: diel vertical migration

### LINKED ISSUE #93: NTR: diel vertical migration
For new term requests, please provide the following information:

## Preferred term label

diel vertical migration

## Textual definition

a circadian behavior of an organism that involves regular movement up and down in the water column in 24 hour cycles. I'm not sure if the "migratory" nature is important here. Is it important to capture that many individuals are moving together?

## Suggested parent term

Circadian behavior

## Attribution

http://orcid.org/0000-0002-2908-3327

### PR DESCRIPTION
**Pull request by DitchingIt**
Fixes #93

### CODE REVIEW

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -9938,6 +9944,27 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
...
+SubClassOf(obo:NBO_0070001 obo:NBO_0000008)
+
+# Class: obo:NBO_0070002 (diel vertical migration)
+
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0070002 "Migration that involves regular movement up and down in the water column.")
+AnnotationAssertion(dc:contributor obo:NBO_0070002 <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(terms:contributor obo:NBO_0070002 <https://orcid.org/0000-0002-4277-2151>)
+AnnotationAssertion(terms:date obo:NBO_0070002 "2023-04-22T14:41:53Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0070002 "behavior_ontology")
+AnnotationAssertion(oboInOwl:source obo:NBO_0070002 "Bandara K, Varpe Ø, Wijewardene L, Tverberg V, Eiane K. Two hundred years of zooplankton vertical migration research. Biological Reviews. 2021 Aug;96(4):1547-89.")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9964]: We usually add references as ISBN, ISBN13 or DOI directly to the definition.

image

File: src/ontology/nbo-edit.owl [hunk 2]
```diff
@@ -9938,6 +9944,27 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070001 (migration)
+
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0070001 "Rhythmic behavior that involves cue-initiated, persistent, direct, focused movement between habitats.")
+AnnotationAssertion(terms:contributor obo:NBO_0070001 <https://orcid.org/0000-0002-4277-2151>)
+AnnotationAssertion(terms:date obo:NBO_0070001 "2023-04-22T14:17:43Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:NBO_0070001 "Movement ecology")
+AnnotationAssertion(oboInOwl:source obo:NBO_0070001 "Dingle H. Migration: the biology of life on the move. Oxford University Press, USA; 2014.")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9953]: See below for a comment on how to add references

**Review by matentzn — CHANGES_REQUESTED** [review]
@DitchingIt looks good to me, but remember, I do not do content - only form - reviews. If you feel confident about your terms and definition, I am happy to trust you. Else I would recommend to find a domain person to look at this.

File: src/ontology/nbo-edit.owl (hunk 1 above)
Comment (DitchingIt) [reply src/ontology/nbo-edit.owl:9964]: 1. I've added the DOI and will make changes to the other PRs.
2. I have had 2 weeks or more suggested changes and conversations for each issue before submitting them as PRs, so I'm not expecting you to review content, nor have I put them forward without the opportunity for comment, but good for you to double-check.

File: src/ontology/nbo-edit.owl [hunk 3]
```diff
@@ -9938,6 +9944,27 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070001 (migration)
+
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0070001 "Rhythmic behavior that involves cue-initiated, persistent, direct, focused movement between habitats.")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9949]: Missing source (DOI/ISBN)

File: src/ontology/nbo-edit.owl [hunk 4]
```diff
@@ -9938,6 +9944,27 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
...
+AnnotationAssertion(terms:contributor obo:NBO_0070001 <https://orcid.org/0000-0002-4277-2151>)
+AnnotationAssertion(terms:date obo:NBO_0070001 "2023-04-22T14:17:43Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:NBO_0070001 "Movement ecology")
+AnnotationAssertion(oboInOwl:source obo:NBO_0070001 "Dingle H. Migration: the biology of life on the move. Oxford University Press, USA; 2014.")
+AnnotationAssertion(rdfs:label obo:NBO_0070001 "migration")
+SubClassOf(obo:NBO_0070001 obo:NBO_0000008)
+
+# Class: obo:NBO_0070002 (diel vertical migration)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI 10.1111/brv.12715") obo:IAO_0000115 obo:NBO_0070002 "Migration that involves regular movement up and down in the water column.")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9959]:
```suggestion
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1111/brv.12715") obo:IAO_0000115 obo:NBO_0070002 "Migration that involves regular movement up and down in the water column.")
```

**Review by matentzn — CHANGES_REQUESTED** [review]

**Review by matentzn — APPROVED** [review]

File: src/ontology/nbo-edit.owl [hunk 5]
```diff
@@ -9963,6 +9964,16 @@ AnnotationAssertion(oboInOwl:source obo:NBO_0070001 "Dingle H. Migration: the bi
 AnnotationAssertion(rdfs:label obo:NBO_0070001 "migration")
 SubClassOf(obo:NBO_0070001 obo:NBO_0000008)
 
+# Class: obo:NBO_0070002 (diel vertical migration)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1111/brv.12715") obo:IAO_0000115 obo:NBO_0070002 "Migration that involves regular movement up and down in the water column.")
+AnnotationAssertion(dc:contributor obo:NBO_0070002 <http://orcid.org/0000-0002-2908-3327>)
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9970]:
```suggestion
AnnotationAssertion(terms:contributor obo:NBO_0070002 <http://orcid.org/0000-0002-2908-3327>)
```

File: src/ontology/nbo-edit.owl [hunk 6]
```diff
@@ -9963,6 +9964,16 @@ AnnotationAssertion(oboInOwl:source obo:NBO_0070001 "Dingle H. Migration: the bi
 AnnotationAssertion(rdfs:label obo:NBO_0070001 "migration")
 SubClassOf(obo:NBO_0070001 obo:NBO_0000008)
 
+# Class: obo:NBO_0070002 (diel vertical migration)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1111/brv.12715") obo:IAO_0000115 obo:NBO_0070002 "Migration that involves regular movement up and down in the water column.")
+AnnotationAssertion(terms:contributor obo:NBO_0070002 <http://orcid.org/0000-0002-2908-3327>)
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9970]:
```suggestion
AnnotationAssertion(terms:contributor obo:NBO_0070002 <https://orcid.org/0000-0002-2908-3327>)
```

**Review by matentzn — APPROVED** [review]

**Pull request merged**